data/net/minecraft/sounds/SoundSource.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/sounds/SoundSource
+	METHOD <init> (Ljava/lang/String;ILjava/lang/String;)V
+		ARG 3 name
```

Comment:
Once again ARG 3 should be ARG 1 ? @sciwhiz12

Reply:
See https://github.com/ParchmentMC/Parchment/pull/18#discussion_r658977910.